src/docs_main.cr

```diff
@@ -62,3 +62,4 @@ require "./uuid"
 require "./uuid/json"
 require "./zip"
 require "./zlib"
+require "./system/*"
```

Comment:
Is there something we're wanting to add to the public API docs? /cc @straight-shoota